aw-server/src/endpoints/key_value.rs

```diff
@@ -0,0 +1,37 @@
+use std::sync::MutexGuard;
+use rocket_contrib::json::Json;
+use rocket::http::Status;
+use rocket::State;
+use crate::endpoints::ServerState;
+
+use aw_datastore::{Datastore, DatastoreError};
+
```

Comment:
Maybe a endpoint for listing all keys?

Reply:
Hm I guess it might be useful? For the first use (storing settings as `key` - `value` pairs) it's probably not necessary but it's pretty low effort to add. I'll look into this.